test/Fail/Issue759a.err

```diff
@@ -1,4 +1,5 @@
 Issue759a.agda:9,13-19
+warning: -W[no]OpenPublicAbstract
```

Comment:
This warning belongs to the class "user might have misunderstood a feature of the Agda language".  (Noted in case we want to find a hierarchical warning scheme.)